pandas/core/series.py

```diff
@@ -4372,6 +4373,34 @@ def between(self, left, right, inclusive=True) -> "Series":
 
         return lmask & rmask
 
+    # ----------------------------------------------------------------------
+    # Convert to types that support pd.NA
+
+    def _convert_dtypes(
+        self: ABCSeries,
```

Comment:
should we either a) not annotate self or b) use "Series" instead of ABCSeries (like we have for the return annotation)

Reply:
When I wrote the code, I didn't know about the `"Series"` annotation, and the return value was caught, so this could be fixed.

@jbrockmendel So now the question is whether these changes are worth a new PR, and whether that could also include doing something with the typing.

Reply:
no worires, ill do this in an upcoming "assorted cleanups" PR